docs/using.rst

```diff
@@ -359,21 +359,21 @@ By default, the following locations are searched:
 Getting help
 ============
 
-If you're having problems you can chat with us on `IRC (#letsencrypt @
-Freenode) <https://webchat.freenode.net?channels=%23letsencrypt>`_ or
-get support on our `forums <https://community.letsencrypt.org>`_.
+If you're having problems you can chat with us on `IRC (#certbot @
+OFTC) <https://webchat.oftc.net?channels=%23certbot>`_ or
```

Comment:
@pde and I are also hanging out in #certbot in OFTC and he mentioned on a call that he preferred OFTC to freenode (for this?)